src/mon/MonCommands.h

```diff
@@ -291,24 +291,20 @@ COMMAND("fs dump "
 COMMAND_WITH_FLAG("mds getmap " \
 	"name=epoch,type=CephInt,req=false,range=0", \
 	"get MDS map, optionally from epoch", "mds", "r", "cli,rest", FLAG(OBSOLETE))
-COMMAND("mds metadata name=who,type=CephString,req=false",
-	"fetch metadata for mds <who>",
+COMMAND("mds metadata name=role,type=CephString,req=false",
+	"fetch metadata for mds <role>",
 	"mds", "r", "cli,rest")
 COMMAND("mds count-metadata name=property,type=CephString",
 	"count MDSs by metadata field property",
 	"mds", "r", "cli,rest")
 COMMAND("mds versions",
 	"check running versions of MDSs",
 	"mds", "r", "cli,rest")
-COMMAND_WITH_FLAG("mds tell " \
-	"name=who,type=CephString " \
-	"name=args,type=CephString,n=N", \
-	"send command to particular mds", "mds", "rw", "cli,rest", FLAG(OBSOLETE))
```

Comment:
This cannot be removed as we get helpful error messages when users do `mds tell`. (That's the reason for `FLAG(OBSOLETE)`.

Reply:
@batrick Done. Please verify. 
Thanks.